.env

```diff
@@ -0,0 +1,2 @@
+FLASK_APP=service:app
+PORT=3000
```

Comment:
I thought this file was in .gitignore 😱

Reply:
Yes, it is. But, since we changed the port number, I think we need this to inform the honcho which port to use.

Reply:
I did a ```git add .env -f``` to add this